codespell_lib/data/dictionary.txt

```diff
@@ -1895,6 +1896,7 @@ appens->appends
 appent->append
 apperance->appearance
 apperances->appearances
+apperant->apparent
```

Comment:
I wonder if this should also suggest the phonetically similar aberrant, like so:
```suggestion
apperant->apparent, aberrant,
```

Reply:
makes sense, adding.